index.css

```diff
@@ -0,0 +1,167 @@
+.grid-container{
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-areas: 
+        'nav-bar'
+        'hero-section'
+        'product-banner'
+        'main-products';
+    height: auto;
+    width: 100vw;
+}
+
+.navigation-bar{
+    grid-area: nav-bar;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    min-height: 8rem;
+    align-items: center;
+    background-color: var(--light-black-background);
+    width: 100vw;
```

Comment:
redundant code. navigation will take the width of the parent container or else you can also give width:100%;

Reply:
yes, made it 100%